nnstreamer_example/custom_example_passthrough/nnstreamer_customfilter_example_passthrough.c

```diff
@@ -51,6 +51,7 @@ pt_init (const GstTensorFilterProperties * prop)
   data->info.dimension[2] = D3;
   data->info.dimension[3] = D4;
   data->info.type = _NNS_UINT8;
+  data->info.layout = _NNS_LAYOUT_NONE;
```

Comment:
This should be always "NONE" if not specified. (specifying it is not incorrect. forcing us to specifying is wrong)

Reply:
Done.